examples/kubernetes/static_provisioning/static_provisioning_with_shared_cache.yaml

```diff
@@ -14,7 +14,7 @@ spec:
   mountOptions:
     - allow-delete
     - region us-west-2
-    - cache-xz s3-csi-driver--usw2-az1--x-s3 # Must be an Express One Zone bucket. More information: https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#shared-cache
+    - cache /tmp/s3-csi-driver-cache # Local cache option for improved performance. More information: https://github.com/awslabs/mountpoint-s3/blob/main/doc/CONFIGURATION.md#caching-configuration
```

Comment:
This is local cache on filesystem level, reused the file from S3 Express One Zone. Still is shared cache